jax/api.py

```diff
@@ -1183,9 +1183,7 @@ def linearize(fun: Callable, *primals) -> Tuple[Any, Callable]:
       or standard Python containers of arrays or scalars. It should return an
       array, scalar, or standard python container of arrays or scalars.
     primals: The primal values at which the Jacobian of `fun` should be
-      evaluated. Should be a tuple of arrays, scalar, or standard Python
-      container thereof. The length of the tuple is equal to the number of
-      positional parameters of `fun`.
+      evaluated, one for each positional argument of `fun`.
```

Comment:
I don't understand the fix: isn't this just erasing the type information that was previously provided? I think the previous version was better.

Reply:
Indeed, I will revert this